libs/wingsdk/src/target-sim/container.inflight.ts

```diff
@@ -55,50 +70,47 @@ export class Container implements IContainerClient, ISimulatorResourceInstance {
       this.managedVolumes = state.managedVolumes;
     }
 
-    // if this a reference to a local directory, build the image from a docker file
-    if (isPath(this.props.image)) {
-      // check if the image is already built
-      try {
-        await runCommand("docker", ["inspect", this.imageTag]);
-        this.log(`image ${this.imageTag} already exists`);
-      } catch {
-        this.log(
-          `building locally from ${this.props.image} and tagging ${this.imageTag}...`
+    if (!(await this.tryInspect(this.imageTag))) {
+      // if this a reference to a local directory, build the image from a docker file
+      if (isPath(this.props.image)) {
+        this.addTrace(
+          `Image ${this.imageTag} not found, building from ${this.props.image}...`,
+          TraceType.RESOURCE
         );
-        await runCommand(
-          "docker",
-          ["build", "-t", this.imageTag, this.props.image],
-          { cwd: this.props.cwd }
+        await this.docker("build", ["-t", this.imageTag, this.props.image], {
+          stdout: true,
+          stderr: true,
+        });
+      } else {
+        this.addTrace(
+          `Image ${this.imageTag} not found, pulling...`,
+          TraceType.RESOURCE
         );
+        await this.docker("pull", [this.imageTag], {
+          stdout: true,
+          stderr: true,
+        });
```

Comment:
Should stdout from pull/build be considered verbose?

Reply:
Since these are pretty long operations in many cases (and we don't have great visibility into the start up state, which we will improve), I figured for now they should be INFO so that users will understand what's going on instead of thinking the simulator is stuck. I am going to work with @skyrpex on improving logging and diagnosability later.

Reply:
OK, I made the "VERBOSE" and add an "INFO" message that says pulling/building is happening.